trust-sdk/trust-sdk-specs.md

```diff
@@ -53,48 +89,66 @@ lrzEHpAAIAmoB0Yegc2ZqxtkSkXlYo_TEg4eBrjopGUj9ySxJh6JlfToGqR7yNKzV8cD_yN_jVR5YrVa
 7Tlfb_0eV47yG1Co&id=4
 ```
 
-### Sign Message
+### Sign simple transaction
+
+This command is the simplified version of `sdk_sign`, it accepts a simple `Transaction` object instead of heavier protobuf model
 
 #### Command
-* Command: `sdk_sign_message`
+
+* Command: `sdk_transaction`
 * Parameters:
   * `coin`: BIP44 coin code
-  * `data`: message to sign, caller must encode or (hash it) as hex string
-  * `meta`: dapp metadata. This attribute is optional and encoded as a dictionary. Check how it's encoded in [Dictionary Encoding](#dictionary-encoding).
+  * `to`: recipient address
+  * `amount`: amount in human-readable (unit) format
+  * `action`: default is `transfer`, other action type like `trade` or `delegate` will be implemented in the future
+  * `confirm_type`: `send` or `sign` flag that indicates if the wallet should broadcast the transaction
+  * `asset`: (Optional) universal asset id, see https://developer.trustwallet.com/add_new_asset/universal_asset_id
+  * `from`: (Optional) specifies which account/address to send
+  * `nonce`: (Optional) Custom nonce or sequence
+  * `fee_price`: (Optional) fee price in smallest unit
+  * `fee_limit`: (Optional) fee limit in smallest unit
+  * `meta`: (Optional) transaction data in hex format, Memo or Destination tag
 
 ##### Example
 
 ```shell
-sampleapp://sdk_sign_message?coin=60&data=4fe61e1a9fb1d18a78977ad1e9611e8c546d54743cf2ff1836fc6933df9f1a54&app=trustsdk&callback=sdk_sign_result&id=1
+trust://sdk_transaction?action=transfer&asset=c1001_t0x514910771af9ca656af840dff83e8264ecf986ca&
+to=0x514910771af9ca656af840dff83e8264ecf986ca&amount=0.01&app=sample_app&callback=sdk_sign_result&
+confirm_type=send&nonce=2&from=0xF36f148D6FdEaCD6c765F8f59D4074109E311f0c&meta=memo&fee_limit=1&
+fee_price=1&id=1
 ```
 
 #### Response
 * Parameters:
-  * `signature`: hex encoded signature
+  * `data`: transaction signed as hex string or transaction hash if `confirm_type` is `send`
```

Comment:
Third part users more easy use separate variables. It hard for them identify what we mean and what data in this parameter.

Reply:
I don't think so
- each method has its own context, for example `jsonrpc` always uses `result` key for different methods, you can still wrap more detailed variable on SDK level
- `sdk_sign` already uses this pattern

I think Android can keep current field name? and set extra `data` too, after a few versions